public/app/features/alerting/partials/alert_tab.html

```diff
@@ -8,7 +8,10 @@ <h4 class="section-heading">Rule</h4>
     <div class="gf-form-inline">
       <div class="gf-form">
         <span class="gf-form-label width-6">Name</span>
-        <input type="text" class="gf-form-input width-20" ng-model="ctrl.alert.name" />
+        <input type="text" class="gf-form-input width-20 gf-form-input--has-help-icon" ng-model="ctrl.alert.name"" />
```

Comment:
@dprokop let me know if you have some extra tricks in mind for helping the user make sure they're formatting a template variable correctly. I thought about adding an `ng-blur` hook to these inputs to check in the case they have tried to use a template variable (search for `"{{"`) and then do a full regex validation to verify it's correctly written - something like `/\{\{\s*\.\w+\s*\}\}/`